docs/user-guide/spark-user-guide.md

```diff
@@ -50,43 +54,46 @@ Usage: SparkCqlEvaluator [options]
     -h, --help
       Print help text
     --input-format
-      Spark SQL format identifier for input files. If not provided, the value
+      Spark SQL format identifier for input files. If not provided, the value 
       of spark.sql.datasources.default is used.
   * -i, --input-path
-      Key-value pair of resource=URI controlling where Spark should read
-      resources referenced in the context definitions file will be read from.
+      Key-value pair of resource=URI controlling where Spark should read 
+      resources referenced in the context definitions file will be read from. 
```

Comment:
i think this should say 'Key-value pair of .... where Spark should read resources referenced in the context definitions file from.   "will be read from"  is redundant.